Sources/Bugsnag/BugsnagConfig.swift

```diff
@@ -0,0 +1,39 @@
+import Vapor
+
+public struct BugsnagConfig {
+    public enum Error: Swift.Error {
+        case invalidConfig(field: String, Config)
+    }
+    
+    public let apiKey: String
+    public let notifyReleaseStages: [String]
+    public let endpoint: String
+    public let filters: [String]
+    public let stackTraceSize: Int
+    public let environment: Environment
+    
+    public init(_ config: Config) throws {
+        apiKey = try config.get("apiKey")
+        
+        notifyReleaseStages = try config["notifyReleaseStages"]?.array?.map {
```

Comment:
This was intentionally an optional since when not set it will always report, while when empty it won't report anything.